calculators/referendum_supporters_calculator.rb

```diff
@@ -29,6 +39,37 @@ def fetch
       end
     end
 
+    late_expenditures.each do |row|
+      sup_opp_cd = guess_whether_committee_supports_measure(row['Filer_ID'], row['Bal_Name'])
```

Comment:
seems like if we can't guess then we drop it.  Its not likely  to happen too often.
On a related note, I think I saw the same ballot measure referred by two different names.